<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrilesM</class>
 <widget class="QWidget" name="FrilesM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文件传输自由「学生端」</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pic.qrc">
    <normaloff>:/Friles.ico</normaloff>:/Friles.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget
{
font: 75 11pt &quot;等线&quot;;
}
QScrollArea {
    border: 0px solid;
    border-right-width: 1px;
    border-right-color: #dcdbdc;
    background-color: #f5f5f7;
}

QScrollBar:vertical {
    border: none;
    background: #f5f5f7;
    width: 10px;
    margin: 0px 0 0px 0;
}

QScrollBar::handle:vertical {
    background: #a68c70;  /* 浅褐色 */
    min-height: 20px;
    border-radius: 5px;
    border: none;
}

QScrollBar::handle:vertical:hover {
    background: #8b7355;  /* 深褐色带一点灰 */
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: 0px solid grey;
    background: #32CC99;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover, QScrollBar::add-line:vertical:hover {
    background: #4ca366;  /* 鼠标悬浮时的颜色 */
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
    width: 0px;
    height: 0px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="styleSheet">
        <string notr="true">border: one;
image: url(:/Friles.ico);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="class_lbl_2">
       <property name="text">
        <string>文件传输自由「学生端」</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="windowModality">
      <enum>Qt::NonModal</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="backSpace_btn_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>上一级</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Refresh_btn_2">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="class_lbl">
       <property name="text">
        <string>高一（13）班</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="goHome_btn_2">
       <property name="text">
        <string>主页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setting_btn">
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pathBar_hl">
     <item>
      <widget class="QLabel" name="subject_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentPath_lbl">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="pathBar_hs">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="fileList_trw">
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border: 0px; </string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustIgnored</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>文件名</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>打开</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>下载</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="goHome_btn">
       <property name="text">
        <string>主页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backSpace_btn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>上一级</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="maximize_btn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>最大化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Refresh_btn">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quit_btn">
       <property name="styleSheet">
        <string notr="true">border: one;
image: url(:/resources/images/quit1.svg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="information_btn">
       <property name="styleSheet">
        <string notr="true">border: one;
image: url(:/resources/images/information.svg);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;等线&quot;;</string>
       </property>
       <property name="text">
        <string>下载网址：combox.top  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;等线&quot;;</string>
       </property>
       <property name="text">
        <string>made by 高2026届</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="HXH_Btn">
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;等线&quot;;border: 0;</string>
       </property>
       <property name="text">
        <string>胡轩豪</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
